core/src/main/java/org/mskcc/cbio/portal/dao/DaoCancerStudy.java

```diff
@@ -317,6 +318,7 @@ public static void addCancerStudy(CancerStudy cancerStudy, boolean overwrite) th
             //data loading process can set this to AVAILABLE:
             //TODO - use this field in parts of the system that build up the list of studies to display in home page:
             pstmt.setInt(10, Status.UNAVAILABLE.ordinal());
+            pstmt.setDate(11, java.sql.Date.valueOf(LocalDate.now()));
```

Comment:
I think our standard pattern is to use import statements instead of full class package specifiers in function code blocks. I guess in this case we have a collision of class names? .. I see - with java.util.Date. It is good that we are using full package specifiers consistently for both classes. I'd probably drop the import "java.util.*" statement too, but it is ok so long we don't rely on it. (Maybe this is why google forbids importing with wildcards)